### DISCUSSION TITLE
Is there any plan to release the latest version of @urql/exchange-request-policy?

### DISCUSSION THREAD

**Discussion by yoshi9930** [urql-graphql/urql#3431] [Q&A]
The version of @urql/core that depends on version 1.0.2 is 4.0.x. Are there any plans to release a version that depends on 4.1.x?

**Comment by JoviDeCroock** [comment]
Latest depends on 4.1.x https://github.com/urql-graphql/urql/blob/main/exchanges/request-policy/package.json#L55 and is released https://www.runpkg.com/?@urql/exchange-request-policy@1.0.2/package.json#45. This because >= 4.0.0 means 4.x.x

**Comment by yoshi9930** [comment]
Thanks for the quick reply.
